serverlessworkflow/antora.yml

```diff
@@ -71,29 +71,29 @@ asciidoc:
     #
     #  Versions 
     #
-    quarkus_version: 3.8.4
-    quarkus_platform_version: 3.8.4
+    quarkus_version: 3.15.3.1
+    quarkus_platform_version: 3.15.3.1
     java_min_version: 17+
-    maven_min_version: 3.9.3
+    maven_min_version: 3.9.6
     graalvm_min_version: 22.3.0
     spec_version: 0.8
     vscode_version: 1.67.0
-    kn_cli_version: 0.32.0
-    kn_cli_tag: 0.32
+    kn_cli_version: 10.0.0
+    kn_cli_tag: 10.0.0
     docker_min_version: 20.10.7
     docker_compose_min_version: 1.27.2
     kubernetes_version: 1.26
-    openshift_version_min: 4.13
+    openshift_version_min: 4.14
     openshift_version_max: 4.17
     knative_version: 1.15
     knative_serving_version: 1.15
     knative_eventing_version: 1.15
-    apache_kie_latest_version: 10.0.0
-    kogito_version: 999-SNAPSHOT
+    apache_kie_latest_version: 10.1.0
+    kogito_version: 10.0.0
     product_version_short: 10.0
     product_version_long: 10.0.0
     # only used in downstream
-    operator_version: main
+    operator_version: 10.0.0
```

Comment:
Don't we have to upgrade these 10.0.0?

Reply:
I  am very confused with myself, ofcit should be `10.1.0` :D